pkg_resources/__init__.py

```diff
@@ -1636,7 +1636,7 @@ def _validate_resource_path(path):
         )
 
     def _get(self, path) -> bytes:
-        if hasattr(self.loader, 'get_data'):
+        if self.loader and hasattr(self.loader, 'get_data'):
```

Comment:
```suggestion
        if hasattr(self.loader, 'get_data') and self.loader:
```
Micro-optimisation. `hasattr` was already safe enough at runtime. truthyness of `self.loader` would never be checked, but satisfy static type-checking.